memeolist/app/src/main/java/org/aerogear/android/app/memeolist/sdk/SyncClient.java

```diff
@@ -20,12 +20,13 @@
 
     private final ApolloClient apolloClient;
 
-    public SyncClient(@Nonnull OkHttpClient okHttpClient, @Nonnull String serverUrl,
+    public SyncClient(QueryStoreWrapper queryStoreWrapper, @Nonnull OkHttpClient okHttpClient, @Nonnull String serverUrl,
                       @Nonnull String webSocketUrl) {
-        apolloClient = ApolloClient.builder().serverUrl(nonNull(serverUrl, "serverUrl"))
+        ApolloClient.Builder builder = ApolloClient.builder().serverUrl(nonNull(serverUrl, "serverUrl"))
                 .okHttpClient(nonNull(okHttpClient, "okHttpClient"))
-                .subscriptionTransportFactory(new Factory(webSocketUrl, okHttpClient))
-                .build();
+                .subscriptionTransportFactory(new Factory(webSocketUrl, okHttpClient));
+        queryStoreWrapper.create(builder, "memeo", "id");
```

Comment:
I think It should be not here since it will need the `dbName` and `cacheKey`. Maybe a method like `getQueryStoreWrapper(String dbName, String cacheKey)`

Reply:
What you mean? Generally we should not expose client in our SDK but builder that will allow users to also customize stuff if needed. I have done that in separate branch.

Reply:
I mean, how we are going to `create` a `QueryStoreWrapper` without knowing the `dbName` and `cacheKey`. How will the dev inform it?

Reply:
That's why I have in memolist. The way it will work is that our SDK will provide builder. I think that db-name it is not significant and it will be "aerogear-sync-db". While key is still unclear depending on if I will manage people to convince to change aproach with the datasources.

Reply:
Let's move on so and revisit it later

Reply:
Keep in mind that this is out of the SDK and once we get complete list of the requirements we can think of how effectively move things into SDK.

Reply:
I see that is why I said, let's move forward ;)